survey/HTML-syntax-in-EPUB.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Survey on the Impact of Allowing HTML Syntax in EPUB
+date: 2025-05-08
+---
+
+The Publishing Maintenance Working Group (PMWG) plans to allow the [HTML syntax](https://html.spec.whatwg.org/multipage/syntax.html#syntax) for the HTML content document format in EPUB 3.4, instead of restricting it to the [XML Syntax](https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax) as it was done for earlier versions of EPUB 3.
+
+This initiative aims to address the diminishing use and maintenance of the XML syntax outside the publishing industry while leveraging the future evolution of HTML. 
+
+The change does not affect the document formats currently allowed in an EPUB publication; they will remain valid as they are today. This change merely removes the restriction to the XML syntax.
+
+PMWG is seeking your valuable feedback.  The purpose of the survey is to get the temperature of the community about this technical change, figure out the potential difficulties, and what the Working Group can do to make the transition easier.  Your insights will help shape the future of EPUB standards. 
+
+Your responses will be kept anonymous but will be made public to ensure transparency. Providing some context about your tools and workflows will help us better understand and address your feedback.
+
+**Question 1:** What is your position in the industry and how do your workflows involve EPUB?
+
+**Question 2:** What potential benefits and challenges do you anticipate when incorporating HTML syntax into your processes? 
+
+**Question 3:** Are there any technical, operational, or resource related issues that need to be considered?
```

Comment:
Should we have a "Q4: Additional comments [optional]"?

Reply:
Yep, that is a good idea. It shows openness...